app/daemon.js

```diff
@@ -101,24 +175,42 @@ export function ensuresIPFSInitialised () {
   })
 }
 
+export function ensureAddressesConfigured () {
+  return getApiMultiAddress()
+    .then(apiMultiAddr => {
+      const isDefaultAddr = apiMultiAddr && apiMultiAddr.indexOf(ORION_API_PORT)
+
+      if (isDefaultAddr) {
+        return setOrionAddresses()
+      }
+
+      return Promise.resolve()
+    })
+}
+
 /**
- * Returns the multiAddr usable to connect to the local dameon via API
+ * 5001 -> 5101
+ * 4001 -> 4101
+ * 8080 -> 8180
  */
-export function getMultiAddrIPFSDaemon () {
-  // Other option: ask the binary wich one to use
-  // const binaryPath = getPathIPFSBinary()
-  // const multiAddr = execSync(`${binaryPath} config Addresses.API`)
-  return '/ip4/127.0.0.1/tcp/5001'
+export function setOrionAddresses () {
+  console.log('Setting Orion custom addresses for API, Gateway and Swarm')
+  const binaryPath = getPathIPFSBinary()
+  const ipfsRepoPath = `IPFS_PATH=${getIPFSRepoPath()}`
+
+  return exec(`${ipfsRepoPath} ${binaryPath} config Addresses.API /ip4/127.0.0.1/tcp/${ORION_API_PORT}`)
```

Comment:
Shouldn't we change the default ports to the new ones only if the standard ones are taken?
What do you say?